### ISSUE TITLE
Typo when getting an error for declaring a module inside a module

### ISSUE THREAD

**Issue by hironichu** [denoland/deno_lint#1004]
### Lint Name

prefer-namespace-keyword

### Code Snippet

```ts
declare module "test" {
	module test {
		export var x: number;
	}
}
```

### Expected Result

`module` keyword in module decl**a**ration is not allowed

### Actual Result

`module` keyword in module decl**e**ration is not allowed

### Version

deno 1.18.1 (release, x86_64-pc-windows-msvc)
v8 9.8.177.6
typescript 4.5.2
